lumicks/pylake/kymotracker/detail/calibrated_images.py

```diff
@@ -0,0 +1,114 @@
+import matplotlib.pyplot as plt
+
+
+class CalibratedKymographChannel:
+    """A kymograph image along with its calibration"""
+
+    def __init__(self, name, data, time_step, pixel_size):
+        self.name = name
+        self.data = data
+        self.time_step = time_step
+        self._pixel_size = pixel_size
+
```

Comment:
I've renamed `time_step` to `time_step_ns`?

For the coordinates, it will depend. Soon we will have to support multiple coordinate systems (sequence position etc), so I'd rather not encode it in the name there.

I've also added a little docstring.

Reply:
fair enough. I think we will need to add some internal attribute to store the calibration type (um, bp, etc...) but probably best to hold off on that until we actually have to implement another spatial coordinate

Reply:
Yeah, I'm in favor of that.  It's also not unlikely that `from_coord` may have two coordinates in the future (if we have dynamic calibrations); but I didn't want to scope creep this PR too much by already introducing the other units.